.github/workflows/lint.yml

```diff
@@ -12,7 +10,7 @@ jobs:
       - name: Set up Go
         uses: actions/setup-go@v3
         with:
-          go-version: "1.18"
+          go-version: "1.19"
```

Comment:
Since we still run our tests on 1.18 maybe it would make sense to continue running linter an govulncheck on 1.18 as well. Or do you think matching these versions doesn't matter?

Reply:
I don't think it makes much difference. I purposefully put the linter to 1.19 as there are new godoc features/rules in 1.19 that I wanted to start enforcing in this repo.